examples/blob2d/blob2d.cxx

```diff
@@ -128,7 +128,7 @@ class Blob2D : public PhysicsModel {
     // Density Evolution
     /////////////////////////////////////////////////////////////////////////////
 
-    ddt(n) = -bracket(phi,n,BRACKET_ARAKAWA)    // ExB term
+    ddt(n) = -bracket(phi,n,BRACKET_SIMPLE)    // ExB term
```

Comment:
Reverted to SIMPLE because that was what was used in v3.1. The ARAKAWA bracket has less dissipation, so results in more noisy solutions which seems less robust than SIMPLE. Since this is one of the easier examples to start with, I thought it better to have it as robust as possible.